packages/server/src/migrations/functions/tests/tableSettings.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { App, Screen, ScreenProps } from "@budibase/types"
+
+const { db: dbCore } = require("@budibase/backend-core")
+const TestConfig = require("../../../tests/utilities/TestConfiguration")
+const migration = require("../tableSettings")
+
+// Local type for allowing any child components inside screens
+type MigratingScreenProps = ScreenProps & { _children: any[] }
```

Comment:
Should `_children` be added to the base `ScreenProps` type? if it's not something that exists only in this migration that should probably be the case

Reply:
`ScreenProps` already has `_children` defined as `ScreenProps[]`, which I'm guessing is because `ScreenProps` is at least more specific than just calling it `any`, but it's not actually correct as it's basically stating that every child of a screen has container settings (`size`, `gap`, `direction`) etc and no others.

I've just changed it to `any` for the test here since I'm adding tables and table blocks to the screen. I guess we don't use this type in the real app otherwise it would break.

This basically comes back to what you said before about having actual component types would be good - but I would prefer not to scope creep this PR by including the introduction of component types since I'm sure that would require quite a few changes.

I'm happy up date `ScreenProps` to change the `_children` property to `any[]` since that's technically correct for now, if you think that's better than just using a temporary type here.

Reply:
Sounds good!